content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -21,29 +21,29 @@ The lifetime of a session involves the following phases:
    4. Destroy the session when the condition is met. For example, when the last player leaves the session. 
     
 
-## 2. Enable AWS GameLift
+## 2. Enable the AWS GameLift Gem
 
-To enable AWS GameLift in your project:  
-  - [Enable AWS GameLift Gem and its dependencies](#enable-aws-gamelift-gem-and-its-dependencies)
-  - [Include AWS GameLift static library](#include-aws-gamelift-static-library)
 
 ### Dependencies
 
 The AWS GameLift Gem depends on the following Gems:
 
 - [AWS Core Gem](/docs/user-guide/gems/reference/aws/aws-core): Provides the framework to use AWS services in O3DE.
-- [Multiplayer Gem](/docs/user-guide/gems/reference/multiplayer): Provides the multiplayer functionality, like connection and hosting, by extending networking framework.
+- [Multiplayer Gem](/docs/user-guide/gems/reference/multiplayer/multiplayer/): Provides multiplayer functionality, like connection and hosting, by extending the networking framework.
   
 
-### Enable AWS GameLift Gem and its dependencies
+### Enable the AWS GameLift Gem and its dependencies
+
 To enable the AWS GameLift Gem in your project:
 1. Open the **Project Manager**. 
 2. Open the menu under your project and choose **Edit Project Settings...**.
-3. Select the **Configure Gems** button.
+3. Choose **Configure Gems**.
 4. Enable the AWS GameLift Gem and the dependent Gems.
 
-### Include AWS GameLift static library
-You must include AWS GameLift static library in your project's CMake build target.
+
+### Include the AWS GameLift Gem static library
+
+You must include the AWS GameLift Gem static library in your project's CMake build target.
 
 1. **(Required)** You must include **Gem::AWSGameLift.Server.Static** as **BUILD_DEPENDENCIES** for your project server target.
 
```

Comment:
In step 2 below this (not in your changes), change "AWS GameLift" ("to make AWS GameLift requests") to "GameLift" to correct the AWS service name.